.github/workflows/tag.yml

```diff
@@ -0,0 +1,26 @@
+name: Tag
```

Comment:
Is this required? 

AFAIU, @negz didn't include this workflow in the functions (e.g. https://github.com/crossplane/function-template-go) to keep things simple, e.g. in favor tagging directly from Github UI.

Reply:
I can see this as helpful in terms of being consistent with other extension repositories, though.

Reply:
Yes, I'd rather not have the tag workflow for functions. I haven't found it to be needed. As @turkenh mentioned, it's just as easy to create a tag from the GitHub UI when cutting a release.